app/controllers/PaymentError.scala

```diff
@@ -0,0 +1,15 @@
+package controllers
+
+sealed trait PaymentError {
+  def message: String
+}
+
+object PaymentError {
+  val allErrors: Map[String, PaymentError] = Map(PaypalError.toString -> PaypalError)
+
+  def fromString(errorCode: String): Option[PaymentError] = allErrors.get(errorCode)
+}
+
+case object PaypalError extends PaymentError {
+  override def message = "Your contribution using PayPal could not be processed. Please try again or use pay by card."
```

Comment:
Your contribution using PayPal could not be processed. Please try again or pay by card.